maturin/__init__.py

```diff
@@ -40,7 +40,15 @@ def _build_wheel(
 ):
     # PEP 517 specifies that only `sys.executable` points to the correct
     # python interpreter
-    command = ["maturin", "pep517", "build-wheel", "-i", sys.executable]
+    command = [
+        "maturin",
+        "pep517",
+        "build-wheel",
+        "-i",
+        sys.executable,
+        "--compatibility",
+        "linux",
```

Comment:
Should this be `"off"` rather than `"linux"`?

Reply:
They are the same right now, but I agree `off` is better.